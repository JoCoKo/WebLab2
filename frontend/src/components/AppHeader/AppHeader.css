.App-header {
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: flex-start;
    font-size: calc(31px);
    color: rgb(111,130,137);
}

.App-header button{
    margin-left: 250px;
    background: rgb(201,210,215);
    font-size: 13px;
    color: inherit;
    text-decoration: none;
    height: 53px;
    border-radius: 10px;
    width: 260px;
}