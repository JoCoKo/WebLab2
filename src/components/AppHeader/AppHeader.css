.App-header {

    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: space-around;
    font-size: calc(31px);
    color: rgb(111,130,137);
}

button{
    background: rgb(201,210,215);
    font-size: 13px;
    color: inherit;
    text-decoration: none;
    height: 53px;
    border-radius: 10px;
    outline: 0;
    border:0;
    cursor: pointer;
}