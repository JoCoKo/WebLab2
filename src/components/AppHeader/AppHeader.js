import React from "react";
import "./AppHeader.css";

const AppHeader = ({updateCity}) => {
    return (
        <header className="App-header">
            <p>Погода здесь</p>
            <button onClick={() => updateCity()}>
                Обновить геолокацию
            </button>
        </header>
    );
};

export default AppHeader;