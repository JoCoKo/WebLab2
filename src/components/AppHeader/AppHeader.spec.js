import React from 'react';
import AppHeader from "./AppHeader";
import {shallowToJson} from "enzyme-to-json";
import {shallow} from "enzyme";

describe('AppHeader', () => {
  const appHeader = shallow(<AppHeader updateCity={() => {}} />);
  it ('отображается корректно', () => {
    expect(shallowToJson(appHeader)).toMatchSnapshot();
  });
});