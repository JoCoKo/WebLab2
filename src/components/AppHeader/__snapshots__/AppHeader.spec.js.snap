// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader отображается корректно 1`] = `
<header
  className="App-header"
>
  <p>
    Погода здесь
  </p>
  <button
    onClick={[Function]}
  >
    Обновить геолокацию
  </button>
</header>
`;
